notebooks/multimodal_vdj/01_preprocessing.ipynb

```diff
@@ -0,0 +1,3169 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # The collumn 'productive' loaded with mixed type (boolean and string), the following line will correct this.
</pre><p>typo column</p><p> </p><p>Maybe a comment like "The column 'productive' contains mixed data types which are not compatible with downstream tools. We correct them by casting them to strings." is better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>